### PR TITLE
Add an initial "working group abilities" document.

### PR DESCRIPTION
**Pull request by dlorenc**
This builds on @dcmiddle's draft at #44. I tried to simplify out the parts that were closer to consensus from that discussion to get something started.

I tried to not bring up the distinction between Project and Working Group here until we get a better idea on why we need to have one.

### CODE REVIEW

File: working-group-abilities.md
Comment (kaywilliams) [inline working-group-abilities.md:25]: From the GB level, we will most likely want all external communications (announcements, blogs, papers, etc.) released under the OpenSSF name to go through review and approval by the GB Communications Committee (not yet formed), This is for consistency and alignment with our overall communications strategy.

File: working-group-abilities.md
Comment (kaywilliams) [inline working-group-abilities.md:37]: The OpenSSF does now have funding.  Let's switch up this section to describe how to request resources.

File: working-group-abilities.md
Comment (kaywilliams) [inline working-group-abilities.md:20]: Can we have some lightweight process for WGs to request approval for creation of new projects?  This is for visibility and alignment across working groups (which might be contemplating similar projects). Perhaps the process is just sending an email to the TAC (and WG Leads?) and asking for 2 individuals to give a 'thumbs up'.  Something for the TAC to discuss.  This aim is not to hamper creativity or progress (both are loved in the OpenSSF!), but to provide ample opportunity for collaboration.

File: working-group-abilities.md
Comment (dlorenc) [reply working-group-abilities.md:25]: I'm happy to try to clarify the scope here, but I'm -1 on requiring technical whitepapers from WGs to be reviewed by the governing board.

File: working-group-abilities.md
Comment (dlorenc) [reply working-group-abilities.md:37]: Any suggestions? Should I just drop this now? I don't think we have that process and don't want to hold up the rest of this on that detail.

File: working-group-abilities.md
Comment (dlorenc) [reply working-group-abilities.md:20]: I'd prefer to leave this up to Working Groups to do on their own. A "firehose" list to announce these too would be fine with me, but working groups need to be able to create repos to do their work.

File: working-group-abilities.md
Comment (kaywilliams) [reply working-group-abilities.md:37]: I'm OK with dropping.

Comment (dlorenc) [reply working-group-abilities.md:36]: Reworded a bit.

File: working-group-abilities.md
Comment (mayakacz) [inline working-group-abilities.md:3]:
```suggestion
Established working groups in the OpenSSF are given broad latitude to accomplish their goals.
```

File: working-group-abilities.md
Comment (mayakacz) [inline working-group-abilities.md:26]: Should likely specify where. In their repo? In the mailing list?

File: working-group-abilities.md
Comment (mayakacz) [inline working-group-abilities.md:33]:
```suggestion
organizations, or to request funding from the OpenSSF Governing Board, through the TAC.
```

**Review by mayakacz — APPROVED** [review]

File: working-group-abilities.md
Comment (kimsterv) [reply working-group-abilities.md:26]: Mailing list preferred but both are ok. A lot of people start with docs first before converting.

**Comment by kimsterv** [comment]
+1 approval (non-binding)

**Comment by dcmiddle** [comment]
We don't have coding standards yet, but I like to have consistent word wrapping. It makes diffs easier for subsequent reviews. For markdown files I think 100 characters is common.

File: working-group-abilities.md
Comment (dcmiddle) [inline working-group-abilities.md:15]: Logistically, I think the working groups will need to request repos from an ossf github org admin (someone in LF).
I don't think github has a mult-tier organization feature that would allow a more autonomous process, i.e. working groups have their own organizations under ossf governance. Similarly there might not be useful mechanics yet to enable working groups to create additional github organizations that are connected to ossf.
So long winded way of saying we should strike "working groups can also create new organizations where necessary" :)

Comment (dlorenc) [reply working-group-abilities.md:15]: > Logistically, I think the working groups will need to request repos from an ossf github org admin

Right now this is actually pretty much everyone in the org - all members can create public repos:
image

I'd be in favor of restricting this to WG leads if we want to later on.

> Similarly there might not be useful mechanics yet to enable working groups to create additional github organizations that are connected to ossf.

For orgs, the CVE benchmark project setup another org already over here: https://github.com/ossf-cve-benchmark. The flow actually went in reverse - they created the org and then added Lindsay to it to transfer to LF.

Maybe we can change that sentence to:

> Working groups can also request new organizations where necessary - reach out to the TAC for logistics.

File: working-group-abilities.md
Comment (dcmiddle) [reply working-group-abilities.md:20]: I'm not sure if Projects and repos are conflated in this comment thread.

There should be one process for Technical Initiatives (Projects and Working Groups) which goes through the TAC. They might be recommended by a Working Group but only the TAC approves through the established process.

The use of the word "projects" here...

> If any of these projects grow large enough, they can be "spun-off" into separate working groups by
> the TAC.

... might be confusing. Can we replace that with "efforts" or some other word that isn't as overloaded as project/Project?

Reserved words: Project, Working Group, Technical Initiative  :)

File: working-group-abilities.md
Comment (dcmiddle) [reply working-group-abilities.md:25]: I'm fine with a comms committee reviewing. I think what was intended here was that we don't create a gauntlet of redundant reviews where a paper is reviewed in a WG, then the TAC, then another committee....

I think the spirit is that the WG does their own technical/subject matter review, and then the Comms Committee sanity checks no one is doing vendor promotion and guides how the content goes out according to, as Kay says, the comms strategy.

Comment (dlorenc) [reply working-group-abilities.md:25]: Yes - no gauntlets and separating concerns is what I'm trying to do here.

In my head, here's how it would work:

- The WG itself should do final sign off on the whitepaper, and then it's considered "published'. Similar to this one from the [Identifying Threats](https://github.com/ossf/wg-identifying-security-threats/tree/main/publications/threats-risks-mitigations) group.
- The GB (and eventual communications/marketing committee) would then review and promote the work through blog posts, tweets, press releases etc. Realistically I imagine this coordination would be happening somewhat in parallel.

**Review by JonZeolla — APPROVED** [review]
(non-binding)

**Comment by dlorenc** [comment]
Made another pass with small changes from review feedback!

File: working-group-abilities.md
Comment (dcmiddle) [reply working-group-abilities.md:29]: The new text below looks like it addresses the comms strategy comment well.
Tweaking this sentence might now be more consistent as 
> ... release through the OpenSSF without review or approval by the TAC.

This wording also removes any unintended interpretation that the papers are not reviewed at all. The working groups should apply their own standards and review processes.

File: working-group-abilities.md
Comment (kaywilliams) [inline working-group-abilities.md:30]: Should we provide more clarity on who or how the papers should be shared for feedback and awareness?  Or, do we want to wait until the communications/marketing committee is established to provide guidance?

**Comment by dlorenc** [comment]
In the context of this abilities doc, I also noticed this in our [FAQ](https://openssf.org/#faq):

> Does the GB or TAC control WGs or projects directly?
Neither the Governing Board (GB) nor the Technical Advisory Committee (TAC) is responsible for managing the foundation hosted WGs and projects directly. Instead, the maintainers of those projects manage them; this includes defining the governance process. The GB is responsible for the budget and the TAC the overall technical strategy.

This seems to indicate that WGs can largely do whatever they want, which is largely how we've been operating anyway. With that said, this document should be more about affirming things that they can do, with basic process for how to do it. If things are not listed in here, it shouldn't imply a WG can't do it.

File: working-group-abilities.md
Comment (dlorenc) [reply working-group-abilities.md:30]: I can't think of anything more concrete to recommend, any suggestions?

Comment (kaywilliams) [reply working-group-abilities.md:30]: How about something like this "... the TAC does ask that drafts be shared as early as possible (ideally at least 7 days in advance of publication) for feedback and awareness by sending email to openssf-tac@lists.openssf.org. The TAC can also help arrange visibility the papers via public announcements, blog posts and tweets."

File: working-group-abilities.md
Comment (dlorenc) [reply working-group-abilities.md:36]: Got it! Thanks.

File: working-group-abilities.md
Comment (dlorenc) [reply working-group-abilities.md]: I think I got this in.

**Comment by dcmiddle** [comment]
lgtm

File: working-group-abilities.md
Comment (kaywilliams) [reply working-group-abilities.md:31]: How about adding something like we said for papers, e.g. "For new efforts, WGs are encouraged to send an email to openssf-tac@lists.openssf.org describing the motivation and goals. This increases opportunities for collaboration, visibility and feedback across the broader OpenSSF community."

**Review by lukehinds — APPROVED** [review]
looks good!

File: working-group-abilities.md
Comment (estesp) [inline working-group-abilities.md:13]: **With** this in mind...

**Review by estesp — APPROVED** [review]
LGTM

**Pull request merged**
